docs/download_files.md

```diff
@@ -5,6 +5,8 @@ These files are delivered as a zip file.
 
 For all downloads, sample folders (indicated by the `SCPCS` prefix) contain the files for all libraries (`SCPCL` prefix) derived from that biological sample.
 Most samples only have one library that has been sequenced.
+For multiplexed libraries, the sample folder name will be a comma-separated list of all samples found in the library files that the folder contains.
```

Comment:
```suggestion
If a download includes multiplexed libraries (i.e. samples combined with cell hashing), the sample folder name will include all samples, separated by an underscore, contained in the library present in the sample folder. 
```
I actually don't know if this is much better... but I think we should have something here that describes what we mean by multiplexing. I also think the docs would benefit from having a FAQ on "What is a multiplexed sample". Then we could include a link to that here. We don't need to include that in this PR, but I do think it would be a good idea to have a place where we can actually explain further.